src/main/java/com/ticketbooking/dto/MovieDto.java

```diff
@@ -0,0 +1,35 @@
+package com.ticketbooking.dto;
+
+import java.time.LocalDateTime;
+
+public class MovieDto {
+    private Long movieId;
```

Comment:
Skoro wszystko jest tylko dostępne getami, to czemu nie finalne? I czemu nie lombok?

Reply:
Lombok jest OK. Nie użyłem go, w tym projekcie ale w pracy jak najbardziej tak. Tak pola w tej klasie nadają się na finalne.